proposed/http-message.md

```diff
@@ -550,7 +566,7 @@ interface RequestInterface extends MessageInterface
      * Extends MessageInterface::getHeaderLines() to provide request-specific
      * behavior.
      *
-     * Retrieves a header by the given case-insensitive name as an array of strings.
+     * Retrieves a header by the given case-insensitive name as a string.
```

Comment:
It might be good to repeat some of the verbiage in getHeaderLines() that describes the return value (a concatenated string, yadda, yadda...)